client/packages/system/src/Location/ListView/LocationEditModal/LocationEditModal.tsx

```diff
@@ -123,15 +123,20 @@ export const LocationEditModal: FC<LocationEditModalProps> = ({
       }
     >
       {!isLoading ? (
-        <Grid flexDirection="column" display="flex" gap={2}>
-          <BasicTextInput
+        <Grid
+          flexDirection="column"
+          display="flex"
+          gap={2}
+          alignItems="flex-start"
+        >
+          <BasicTextInput sx={{width: 450}}
```

Comment:
I'd prefer to avoid having a fixed number value for the width - though it requires a bit of fixing of the `BasicTextInput` to make that work 🙄 

Would you mind changing to `fullWidth` instead of the style - and then you can remove the `alignItems` from the `Grid` too.

```suggestion
        <Grid flexDirection="column" display="flex" gap={2}>
          <BasicTextInput
            fullWidth
```

You just have to add this to `BasicTextInput`

```
export const BasicTextInput: FC<BasicTextInputProps> = React.forwardRef(
  ({ sx, InputProps, error, required, ...props }, ref) => (
    <Box
      display="flex"
      justifyContent="flex-end"
      alignItems="center"
      width={props.fullWidth ? '100%' : undefined}
    >
```

that `width` prop needs to be added 😉

Reply:
Thank you @mark-prins , Great tips !
Done by this: [a7449b0](https://github.com/openmsupply/open-msupply/pull/1574/commits/a7449b00ccbeda59ae11e0e7408979e695aa44f7)